src/FirstSteps/RPG/Game.cs

```diff
@@ -32,51 +25,63 @@ public static void CreateHero()
             _hero = HeroesCreator.Create(name, race);
 
         }
-        public static void HandleCommand(string command)
+        private static void HandleCommand(string command)
         {
-            switch (command)
+            var adventure = _adventuresRegistry.GetAdventure(command);
```

Comment:
```csharp
var adventure = _adventuresRegistry.GetAdventure(command);
if(adventure != null)
{
    adventure.Enter(_hero);
}
else if(command == "Stats")
{
    DisplayHeroStats();
}
```
It would be cleaner because, at first look, you see what happens.